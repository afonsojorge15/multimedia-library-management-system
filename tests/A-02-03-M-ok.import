DVD:Era uma vez na Amadora:Fernando Fonseca:20:FICTION:200505550:10
DVD:48 Horas para o Exame:Orlando Fonseca:12:FICTION:200505553:10
DVD:Lumiar Selvagem:Pedro Fonseca:20:FICTION:200505551:5
