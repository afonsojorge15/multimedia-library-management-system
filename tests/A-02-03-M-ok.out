Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: 0 - 10 de  10 - DVD - Era uma vez na Amadora - 20 - Ficção - Fernando Fonseca - 200505550
1 - 10 de  10 - DVD - 48 Horas para o Exame - 12 - Ficção - Orlando Fonseca - 200505553
2 - 5 de  5 - DVD - Lumiar Selvagem - 20 - Ficção - Pedro Fonseca - 200505551
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 